rdtools/analysis_chains.py

```diff
@@ -252,7 +255,8 @@ def _calc_clearsky_poa(self, times=None, rescale=True, **kwargs):
         clearsky_poa = clearsky_poa['poa_global']
 
         if aggregate:
-            interval_id = pd.Series(range(len(self.poa_global)), index=self.poa_global.index)
+            interval_id = pd.Series(
+                range(len(self.poa_global)), index=self.poa_global.index)
```

Comment:
FYI we've adopted a max line length of 99 for rdtools (https://github.com/NREL/rdtools/blob/master/.flake8#L5), so no need to wrap at 80 in this repo